### ISSUE TITLE
Should there be an optional comments field in the Trash Point?

### ISSUE THREAD

**Issue by kadry55** [letsdoitworld/World-Cleanup-Day#127]
Problem description: I just made a trash point of something which looks like a trash but it's an old wooden box. So would like to add that the box should be dislocated to prevent further litter

**Comment by krishaamer** [comment]
I think we had it in the designs made by https://dux.ee/ - not really sure why we didn't implement. @KristiinaKerge do you know?

**Comment by KristiinaKerge** [comment]
Yes we have it in designs, we can add it with notification module, in first dev we had to kill it

**Comment by kristiinapalu** [comment]
1. It should take 4h of development and can be developed with any developer
2. Developer has to add an extra field to the database and app
